commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/tsv/deserialization/TsvDeserializer.kt

```diff
@@ -47,6 +47,7 @@ class TsvDeserializer(private val chunkProcessor: ChunkProcessor = ChunkProcesso
 
     private fun chunkerize(pagetab: String) =
         pagetab.split(TSV_CHUNK_BREAK)
+            .map { it.removeSuffix("\r") }
```

Comment:
can you explain this one please

Reply:
This is related to MacOS end of line encoding. We just need to remove it if present.